lib/core/tests/physics_simulator.rs

```diff
@@ -58,25 +65,25 @@ mod tests {
             steps_per_second,
             properties.clone(),
             move |position| {
-                object.set_position(position);
+                object.mod_position(|t| *t = position);
+                world.display_object().update();
             }
         );
 
         // Updates spring's fixed point every two seconds.
         let every = 2.0;
         let animator  = FixedStepAnimator::new(1.0 / every, move |_| {
+            let _keep_alive = &simulator;
+            let _keep_alive = &css3d_system;
+
             let x = 320.0 * random() as f32;
             let y = 240.0 * random() as f32;
             let z = 0.0;
             let position = Vector3::new(x, y, z);
             properties.mod_spring(|spring| spring.fixed_point = position);
-            target.set_position(position);
+            target.mod_position(|t| *t = position);
         });
 
-        b.iter(move || {
-            let _keep_alive = &simulator;
-            let _keep_alive = &animator;
-            renderer.render(&mut camera, &scene);
-        });
+        std::mem::forget(animator);
```

Comment:
:(

Reply:
Same thing as above. Someone has to be forgotten or it will be dropped.